opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -231,6 +231,7 @@ def create_file(filename: str) -> None:
     """
     global CURRENT_FILE, CURRENT_LINE
     if os.path.exists(filename):
+        open_file(filename)
         raise FileExistsError(f"File '{filename}' already exists.")
```

Comment:
```suggestion
        raise FileExistsError(f"File '{filename}' already exists so it was not newly created, opening existing file.")
```

Reply:
The phrase "so it was not newly created" is inaccurate because FileExistsError can be raised regardless of whether the file was newly created or existed beforehand. It is better to use a concise and accurate message.

Reply:
Sorry, I don't understand this:
> FileExistsError can be raised regardless of whether the file was newly created or existed beforehand

Won't this error only be thrown if the file already exists?

Reply:
Yeah, tbh I feel that in this context "newly created" and "created now" mean the same thing in this context and the former is a bit more natural, but if you think it's confusing I think "created now" could also work.

Reply:
We can ignore that phrase because "File '{filename}' already exists" is much more clear.